R/imputation.R

```diff
@@ -0,0 +1,326 @@
+##' @title Quantitative mass spectrometry data imputation
+##'
+##' @description
+##'
+##' The `impute_matrix` function performs data imputation on `matrix`
+##' objects instance using a variety of methods (see below).
+##'
+##' Users should proceed with care when imputing data and take
+##' precautions to assure that the imputation produce valid results,
+##' in particular with naive imputations such as replacing missing
+##' values with 0.
+##'
+##' @details
+##'
+##' There are two types of mechanisms resulting in missing values in
+##' LC/MSMS experiments.
+##'
+##' - Missing values resulting from absence of detection of a feature,
+##'   despite ions being present at detectable concentrations. For
+##'   example in the case of ion suppression or as a result from the
+##'   stochastic, data-dependent nature of the MS acquisition
+##'   method. These missing value are expected to be randomly
+##'   distributed in the data and are defined as missing at random
+##'   (MAR) or missing completely at random (MCAR).
+##'
+##' - Biologically relevant missing values resulting from the absence
+##'   of the low abundance of ions (below the limit of detection of
+##'   the instrument). These missing values are not expected to be
+##'   randomly distributed in the data and are defined as missing not
+##'   at random (MNAR).
+##'
+##' MNAR features should ideally be imputed with a left-censor method,
+##' such as `QRILC` below. Conversely, it is recommended to use host
+##' deck methods such nearest neighbours, Bayesian missing value
+##' imputation or maximum likelihood methods when values are missing
+##' at random.
+##'
+##' Currently, the following imputation methods are available. 
+##'
+##' - *MLE*: Maximum likelihood-based imputation method using the EM
```

Comment:
would it be more intuitive for the user if you used here `method = "MLE"` instead of *MLE*?

Reply:
But I would need to repeat `method = ` every time. I think, given that method is mentioned before the list, it should be fine.

Reply:
OK